app.py

```diff
@@ -27,49 +30,45 @@
 app = Quart(__name__)
 app.config['TESTING']= False
 
+
 async def get_github_data(code, discord_id):
-    github_url_for_access_token = 'https://github.com/login/oauth/access_token'
-    data = {
-        "client_id": os.getenv("GITHUB_CLIENT_ID"),
-        "client_secret": os.getenv("GITHUB_CLIENT_SECRET"),
-        "code": code
-    }
+   
     headers = {
         "Accept": "application/json"
     }
 
     async with aiohttp.ClientSession() as session:
-        async with session.post(github_url_for_access_token, data=data, headers=headers) as response:
-            r = await response.json()
-            auth_token = (r)["access_token"]
-            headers = {
-                "Authorization": f"Bearer {auth_token}"
+        r = await GithubAdapter.get_github_data(code)
+
+        auth_token = (r)["access_token"]
+        headers = {
```

Comment:
if `headers` is being re-defined here, let's get rid of the definition on on Line#36